apps/client/src/routes/Booking/Booking.tsx

```diff
@@ -259,11 +262,19 @@ export const Booking = () => {
                 component="form"
                 onSubmit={form.handleSubmit(nextStepOrSubmit)}
               >
-                {activeStep === steps.length && vpoUser ? (
+                {activeStep === steps.length && registeredVpoData ? (
                   <BookingInfo
-                    vpoReferenceNumber={vpoUser.vpoReferenceNumber}
-                    bookingDate={vpoUser.scheduleDate}
-                    receivedHelpDate={vpoUser.receivedHelpDate}
+                    vpoReferenceNumbers={[
+                      registeredVpoData.mainVpo.vpoReferenceNumber,
+                    ].concat(
```

Comment:
Array concatenation must be wrapped in useMemo if passed as prop to prevent rerenders.

Reply:
💩

Reply:
done